catalog-info.yaml

```diff
@@ -104,3 +104,48 @@ spec:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-dra-artifacts
+  description: Buildkite pipeline for packaging Elastic Agent core binary and publish it to DRA
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/elastic-agent
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: dra-artifacts
+      description: Buildkite pipeline for packaging Elastic Agent core binary and publish it to DRA
+    spec:
+      pipeline_file: ".buildkite/pipeline.dra-artifacts.yml"
+      provider_settings:
```

Comment:
Let's put the same `provider_settings` as the other pipelines to make sure builds are only triggered by the `elasticmachine`